### PR TITLE
add option to force mavlink_passthrough

### PR DESCRIPTION
**Pull request by DanielePettenuzzo**
I ran out of time so I will test this first thing in the morning.

### CODE REVIEW

File: src/protocol_splitter.cpp [hunk 1]
```diff
@@ -669,6 +669,8 @@ static int parse_options(int argc, char **argv)
 
 		case 'h': usage(argv[0]);			return -1;				break;
 
+		case 'm': _options.force_mavlink_passthrough = true; 		break;
```
Comment (TSC21) [inline src/protocol_splitter.cpp:672]: Can you fix the identations?

File: src/protocol_splitter.cpp [hunk 2]
```diff
@@ -695,12 +697,24 @@ static int parse_options(int argc, char **argv)
 	return 0;
 }
 
+static void parse_env_variables()
```
Comment (TSC21) [inline src/protocol_splitter.cpp:700]: Why do we need this if we can just pass the argument when starting the app then?

**Review by TSC21 — CHANGES_REQUESTED** [review]

File: src/protocol_splitter.cpp (hunk 1 above)
Comment (DanielePettenuzzo) [reply src/protocol_splitter.cpp:672]: Strange. I didn't notice it. Thank you

File: src/protocol_splitter.cpp (hunk 2 above)
Comment (TSC21) [reply src/protocol_splitter.cpp:700]: Well this is a workaround anyway for now. Question is: can we take a more generic env variable name? This a public repo and others will use it so makes more sense to set a second env variable to make it cleaner or have this as a separate branch (not merged to master).

I will anyway create the PR tomorrow morning to make the check more robust (didn't have the time tonight).

**Review by TSC21 — APPROVED** [review]

**Pull request merged**